custom_components/fraimic/image_convert.py

```
@@ -209,6 +229,139 @@ def _preprocess(
+                + diff_b * diff_b * 0.400
+            ) * 0.75 / (255.0 * 255.0)
+            luma_diff = luma - palette_luma[index]
+            distance = 1.5 * rgb_distance + 0.60 * luma_diff * luma_diff
+            better = distance < best_distance
+            best_distance[better] = distance[better]
+            best_index[better] = index
+        lut[red] = best_index
+    return lut
+
+
+def _official_atkinson_indices(image):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_official_atkinson_indices`

(ANN202)